test/bin/stress_testing.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
+
+set -uo pipefail
+
+SCRIPT_NAME="stress_testing.sh"
+VERSION="0.2.0"
+
+INTERFACE="enp1s0" # interface in the MicroShift Jenkins VMs
```

Comment:
You might consider deriving the interface. This example matches on the route with a metric of 100 and the default GW
route | awk '/100/ && /default/{print $8}'

Reply:
thanks, I added a command to get default interface